.pipelines/e2e.yaml

```diff
@@ -16,12 +16,14 @@ pr:
       - .pipelines/templates/e2e-template.yaml
       - e2e
       - parts/linux
+      - parts/windows
       - pkg/agent
+      - staging/cse/windows/
```

Comment:
I am overall fine with this change, but food for thought: now this pipeline which is combined, will be triggered for linux and windows changes too, but for a linux only change, what if it gets blocked because of a non relevant windows test failure given that its a combined pipeline?

Reply:
I set this for linux pipelines
```
TAGS_TO_SKIP: "os=windows"
```

And this for windows:
```
TAGS_TO_RUN: "os=windows"
```

Reply:
The main goal is to switch from bash to golang

Reply:
Il not understanding the ENV. it would be great if we could not have to learn a new inhouse languages to operate the pipeline logic :).  So the skip env are to be changed or they have good value for ever ?

Reply:
Do you have a better idea?